HackYourFuture-CPH/Guide-IT
# Created a card profile result component

# Description

I created a component to display the results of the quiz, to be able to see different results depending on the answers that a user gives in the quiz. This is so that a user can see the guidance bullet points for their specific result.

Fixes # 16
https://github.com/HackYourFuture-CPH/Guide-IT/issues/16

# How to test?

Open the branch and check the component out in storybook. The component is modelled in two ways, as a personal profile and as a professional profile.

# Checklist

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] This PR is ready to be merged and not breaking any other functionality
